core/persistence-jpa-json/pom.xml

```diff
@@ -321,6 +331,16 @@ under the License.
           <version>${jdbc.mysql.version}</version>
           <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
maybe you can add these dependencies in the main profile, to avoid duplication (there are of course no tests run by the main profile)

Reply:
You mean in the root pom under a <dependencies> tag, right?

Reply:
@mmoayyed when applying the diff in https://paste.apache.org/69xl3, I verified that both `-Ppostgres` and `-Pmysql` pass, under `core/persistence-jpa-json` (the diff also includes the fix for comment above).

Reply:
Thanks very much. Unit tests are passing. Moving onto FITs.